### PR TITLE
Implement robotk review command

### LINKED ISSUE #7: Game reviews
# Game Reviews

As a User, I'd like to be able to ask randbot to aggregate reviews for me

### Primary
- Asking for a review for a game should give me results
- Integrate with metacritic
- Integrate with steam (?)
### Secondary
- Asking for a review for a game should give me an averaged score across aggregated results

## Criteria for acceptance
- [ ] Calling `@randbot review [game]` should give me an aggregated list of reviews

### PR DESCRIPTION
**Pull request by paroxp**
## What

We'd like to receive some data from an API regarding a game we're currently
talking about in the chat. Robotk could provide us with that information if we
ask nicely.

`nock` is a testing library to mock any http request. AFAIK it doesn't
make the request, but it captures any directed at this direction, then
composes own response later on.

## How to review

- Code review
- Run tests
- Attempt to ask robotk to obtain review of a game - You'll need to provide `IGDB_API_KEY` env var
- Decide whether the message format provided is sufficient

Closes #7

### CODE REVIEW

File: tests/review.spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  summary: 'splendid',
+  cover: {
+    url: 'http://example.com/img.png'
+  },
+  url: 'http://example.com/'
+}];
+
+describe('robotk review', () => {
+  beforeEach(() => {
+    nock('https://api-endpoint.igdb.com')
```
Comment (mikestephens) [inline tests/review.spec.js]: Unfamiliar with nock. Does it actually send a request?

Comment (paroxp) [reply tests/review.spec.js]: `nock`  is a testing library to mock any http request. AFAIK it doesn't make the request, but it captures any directed at this direction, then composes own response later on.

Probably should have been in commit message. Sorry!

**Comment by mikestephens** [comment]
Looks like CI is red

**Comment by paroxp** [comment]
CI is red for no reason. Could you restart the build?

**Pull request merged**